src/styles.css

```
@@ -155,15 +159,71 @@ body.modal-open{overflow:hidden;  height:100vh;}
+  position:absolute;
+  top:0;left:0;right:0;
+  height:2.5px;
+  background:linear-gradient(90deg,var(--orange),#FBBC04);
+  opacity:0;
+  transition:opacity .2s;
+}
+/* 3D active state — applied by JS on mousemove */
+.org-card.tilt-active{
+  border-color:var(--orange-border);
+  box-shadow:
+    0 2px 4px rgba(28,16,8,.04),
```

Stylelint (17.11.1)

[error] Expected "rgba" to be "rgb" (color-function-alias-notation)

(color-function-alias-notation)